integration/test/Test/Spar.hs

```diff
@@ -338,7 +345,16 @@ testSparCreateScimTokenWithName :: (HasCallStack) => App ()
 testSparCreateScimTokenWithName = do
   (owner, _tid, _) <- createTeam OwnDomain 1
   let expected = "my scim token"
-  createScimTokenWithName owner expected >>= assertSuccess
+  createScimTokenV6 owner (def {name = Just expected}) >>= assertSuccess
   tokens <- getScimTokens owner >>= getJSON 200 >>= (%. "tokens") >>= asList
   for_ tokens $ \token -> do
     token %. "name" `shouldMatch` expected
+
+testCreateMultipleIdps :: (HasCallStack) => App ()
+testCreateMultipleIdps = do
```

Comment:
can this test be removed.

come to think of it, can we remove some tests anywhere?  any new weed?